Code/6-wire-version/src/main.cpp

```diff
@@ -167,7 +213,110 @@ void sendMQTT(){
 
 
 
-void startOTA() { // Start the OTA service
+/**
+ * Start a Wi-Fi access point, and try to connect to some given access points.
+ * Then wait for either an AP or STA connection
+ */
+void startWiFi()
+{
+  //WiFi.mode(WIFI_STA);
+  WiFi.setAutoReconnect(true);
+  WiFi.persistent(true);
+  WiFi.hostname(String("layzspa-" + ESP.getChipId()).c_str());
+
+  if (enableStaticIp4)
+  {
+    Serial.println("WiFi > using static IP \"" + ip4Address.toString() + "\" on gateway \"" + ip4Gateway.toString() + "\"");
+    WiFi.config(ip4Address, ip4Gateway, ip4Subnet, ip4DnsPrimary, ip4DnsSecondary);
+  }
+
+  if (enableAp)
```

Comment:
@torei What's the use case for setting the access point from web gui? You must obviously already be connected to be able to change this. Is it for people with several networks if they want to switch to another SSID? Or to prepare for a new password?

Reply:
yes, thats it. i have a network for my house and one for the garden and garage. i had problems connecting stuff from out of the poolhouse and the garage. i'm not using a mesh wifi so i just created a hidden AP for outside the house.
i was also thinking of a second network to be configured like tasmota is doing.
https://tasmota.github.io/docs/Commands/#wi-fi

Reply:
I see. One could play with the multiwifi library. I think you can have 3 AP:s stored but I've never used it.

Reply:
thanks for the hint.
https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WiFi/examples/WiFiMulti/WiFiMulti.ino